src/constants/collectionCoverImage.ts

```diff
@@ -0,0 +1,42 @@
+export const mainPageCollections = [
```

Comment:
Hmm, maybe it makes sense to move this constants to `domains/collection/constants.ts`
Also, you can consider adding type for the colection: 
```ts
type Collection = {
  title: string
  name: string
  imageUrl: string
}
```

I think `pathName` can be replaced with `name` (for example `dresses` or `rings`), so then you can build any url dynamically:
```tsx
`/items?category=${collections[0].category}`
`/admin/items?category=${collections[0].category}`
```